go/vt/srvtopo/resilient_server.go

```diff
@@ -207,53 +244,94 @@ func (server *ResilientServer) GetSrvKeyspaceNames(ctx context.Context, cell str
 	}
 	server.mutex.Unlock()
 
-	// Lock the entry, and do everything holding the lock.  This
-	// means two concurrent requests will only issue one
-	// underlying query.
+	// Lock the entry, and do everything holding the lock except
+	// querying the underlying topo server.
+	//
+	// This means that unless the topo server is very slow, two concurrent
```

Comment:
I'm not sure about the 'unless the topo server is very slow' part any more, is that still true?

Reply:
Nope. Updated.